internal/consensus/reactor.go

```diff
@@ -1164,6 +1244,8 @@ func (r *Reactor) handleDataMessage(envelope p2p.Envelope, msgI Message) error {
 		return nil
 	}
 
+	logger.Debug("data channel processing", "msg", envelope.Message, "type", fmt.Sprintf("%T", envelope.Message))
```

Comment:
wdyt about to change `fmt.Sprintf("%T", envelope.Message)` on a function that could returns a type structure?
the idea is to avoid using `fmt.Sprintf` that is used `reflect` under the hood.
```
func messageType(msg proto.Message) string {
	switch msg.(type) {
	case *protop2p.PexRequest:
		return "*protop2p.PexRequest"
	default:
		return fmt.Sprintf("%T", msg)
	}
}
```
if the case statement is matched on the type, then to return a predefined string,  otherwise there is a fallback

Reply:
Leaving as it is, this is an overkill as the code is only called max once a second